tox.ini

```diff
@@ -2,19 +2,19 @@
 envlist = python, node, go, openjdk
 isolated_build = True
 
+[tox:.package]
+basepython = python3
+
 [testenv]
-deps =
-    pytest
-    pytest-testinfra
-    # psutil used to detect cpus
-    pytest-xdist[psutil]
-    prettytable
-allowlist_externals = docker
+allowlist_externals =
+    docker
+    poetry
+    podman
 commands =
-    m8a-fetch-language-containers {envname}
+    poetry run m8a-fetch-language-containers {envname}
     # -n auto tells pytest to use as many processors as detected
     # to run all the tests from the module test_{envname} in parallel
-    pytest -n auto test_{envname}.py []
+    poetry run pytest -n auto test_{envname}.py []
 
 [testenv:fetch-all]
 commands =
```

Comment:
Can you update the poetry commands here too?

Reply:
Like this?

Reply:
Yup!